tests/integration/tests/test_smoke.py

```diff
@@ -23,7 +24,7 @@ def test_smoke(instances: List[harness.Instance]):
     )
 
     instance.exec(["k8s", "bootstrap", "--file", bootstrap_smoke_config_path])
-    util.wait_until_k8s_ready(instance, [instance])
+    util.wait_until_k8s_ready(instance, [instance], retries=120, delay_s=10)
```

Comment:
any reason for this large amount of retries?
The cluster should not need 20min to get up, even if it has to pull everything.
This would only take that long if something is broken and then we don't want the runner to execute the test for that long.
Save the planet! :)

Reply:
you're absolutely right :D 
The problem was that during my tests the node sometimes took +10 minutes to get ready, don't know why. I'll change this back to the original values.

Reply:
Hm, the node is marked as ready once the CNI (Cilium) is deployed. This takes around 2min on my local machine.
However, it pulls the images so this heavily depends on your internet connection.

Reply:
yea sometimes the VMs on my machine can not use the full bandwidth, they reach <1MBps at best.